astropy/table/bst.py

```diff
@@ -483,5 +483,5 @@ def replace_rows(self, row_map):
         row_map : dict
             Mapping of row numbers to new row numbers
         """
-        for key, data in self.items():
+        for _, data in self.items():  # noqa: PERF102
```

Comment:
Maybe we should implement `keys()` and `values()` methods for this class, but that's beyond the scope of this pull request.

Reply:
Agreed, on both counts.